docs/src/pages/components/grid/SpacingGrid.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import {
+  makeStyles,
+  createStyles,
+  Theme,
+  createMuiTheme,
+  MuiThemeProvider,
+} from '@material-ui/core/styles';
 import Grid, { GridSpacing } from '@material-ui/core/Grid';
 import FormLabel from '@material-ui/core/FormLabel';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import RadioGroup from '@material-ui/core/RadioGroup';
 import Radio from '@material-ui/core/Radio';
 import Paper from '@material-ui/core/Paper';
 
+const customTheme = createMuiTheme({
+  spacing: factor => `${factor}rem`,
```

Comment:
I don't think that we should change the demo. I have no doubt it was useful for debugging. A test of the generated CSS should be enough. I can help with that.

Reply:
Sure, I can return the standard demo component first. Then I can write some function, that returns generated CSS styles and make a couple of test cases for that function, or something else? If it's ok, I can do this today later.

Reply:
I think that we can use the server side API to assert that the generated styles are correct, https://github.com/mui-org/material-ui/blob/8c94e2a608455baa1c8bf8e93c61d17fe011a55a/packages/material-ui-styles/src/StylesProvider/StylesProvider.test.js#L77.

Reply:
I will need time to deal with this, or your help :)